### PR TITLE
Added storage of intent and slot labels inside of the config and did …

### PR DESCRIPTION
**Pull request by vladgets**
Added storage of intent and slot labels inside of the config and did correction in prediction function

### CODE REVIEW

File: nemo/collections/nlp/models/intent_slot_classification/intent_slot_classification_model.py [hunk 1]
```diff
@@ -289,13 +293,21 @@ def _setup_infer_dataloader(self, queries: List[str], batch_size: int) -> 'torch
...
             dataset=dataset,
             collate_fn=dataset.collate_fn,
             batch_size=batch_size,
             shuffle=False,
-            num_workers=self._cfg.test_ds.num_workers,
-            pin_memory=self._cfg.test_ds.pin_memory,
+            num_workers=num_workers,
+            pin_memory=pin_memory,
             drop_last=False,
         )
```
Comment (tkornuta-nvidia) [inline nemo/collections/nlp/models/intent_slot_classification/intent_slot_classification_model.py:312]: ok, this makes sense 👍

**Comment by okuchaiev** [comment]
@vladgets and/or @tkornuta-nvidia could you please resolve merge conflicts?

**Comment by tkornuta-nvidia** [comment]
Hey, I am not responsible for this PR. All the TLT-related changes to this model that I need were incorporated in #1477 .

@vladgets it is your shot on what you want to do with this.

**Comment by vladgets** [comment]
Since Tomasz did his changes in the code, I will just cancel this PR request.

**Pull request closed without merging**